src/components/application_manager/test/commands/hmi/ui_get_capabilities_response_test.cc

```diff
@@ -267,6 +289,70 @@ TEST_F(UIGetCapabilitiesResponseTest, SetPhonenCapability_SUCCESS) {
   command->Run();
 }
 
+TEST_F(UIGetCapabilitiesResponseTest, SetVideoStreamingCapability_SUCCESS) {
+  MessageSharedPtr command_msg = CreateCommandMsg();
+  (*command_msg)[strings::msg_params][strings::system_capabilities] =
+      smart_objects::SmartObject(smart_objects::SmartType_Map);
+
+  (*command_msg)[strings::msg_params][strings::system_capabilities]
+                [strings::video_streaming_capability] =
+                    smart_objects::SmartObject(smart_objects::SmartType_Map);
+
+  (*command_msg)[strings::msg_params][strings::system_capabilities]
+                [strings::video_streaming_capability]
+                [strings::preferred_resolution] =
+                    smart_objects::SmartObject(smart_objects::SmartType_Map);
+
+  (*command_msg)[strings::msg_params][strings::system_capabilities]
+                [strings::video_streaming_capability]
+                [strings::preferred_resolution][strings::resolution_width] =
+                    800;
+
+  (*command_msg)[strings::msg_params][strings::system_capabilities]
+                [strings::video_streaming_capability]
+                [strings::preferred_resolution][strings::resolution_height] =
+                    350;
+
+  (*command_msg)[strings::msg_params][strings::system_capabilities]
+                [strings::video_streaming_capability][strings::max_bitrate] =
+                    10000;
+
+  (*command_msg)[strings::msg_params][strings::system_capabilities]
+                [strings::video_streaming_capability]
+                [strings::supported_formats] =
+                    smart_objects::SmartObject(smart_objects::SmartType_Array);
+
+  (*command_msg)[strings::msg_params][strings::system_capabilities]
```

Comment:
@shoamano83 I would extract (*command_msg)[strings::msg_params][strings::system_capabilities]  [strings::video_streaming_capability] to some reference for reducing lines of code
```
SmartObject& video_streaming_capability = (*command_msg)[strings::msg_params][strings::system_capabilities]  [strings::video_streaming_capability];
...
```